submitter/jobsubmitter.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, job_id, app, inputs, root_dir):
         self.lsf_client = LSFClient()
         self.job_store_dir = os.path.join(settings.TOIL_JOB_STORE_ROOT, self.job_id)
         self.job_work_dir = os.path.join(settings.TOIL_WORK_DIR_ROOT, self.job_id)
-        self.job_outputs_dir = os.path.join(root_dir, 'outputs')
+        self.job_outputs_dir = os.path.join(root_dir)
```

Comment:
Why not just self.job_outputs_dir = root_dir?

Reply:
Because deleting it was faster and I was lazy

Reply:
And I didn't think about it

Should be fixed now.